grails-app/views/order/_orderItemFormDialog.gsp

```diff
@@ -15,6 +15,9 @@
                     <format:product product="${orderItem.product}"/>
                     <g:hiddenField id="dlgProduct" name="product.id" value="${orderItem?.product?.id}"/>
                     <g:hiddenField id="dlgSupplierId" name="supplier.id" value="${orderItem?.order?.originParty?.id }"></g:hiddenField>
+                    <g:hiddenField id="hasBudgetCodeSupport" name="hasBudgetCodeSupport"
```

Comment:
@DominikaHatala I think, that this should have a different name. Now it is suggesting that the whole feature might be disabled/enabled, but in reality, it is indicating if the field is required. I suggest `isBudgetCodeRequired`.